terraso_backend/apps/graphql/schema/audit_logs.py

```diff
@@ -0,0 +1,58 @@
+import graphene
+from django_filters import FilterSet, OrderingFilter
+from graphene import relay
+from graphene.types.generic import GenericScalar  # Solution
+from graphene_django import DjangoObjectType
+
+from apps.audit_logs import models
+
+from .commons import TerrasoConnection
+
+
+class AuditLogNode(DjangoObjectType):
+    """
+    AuditLogNode is a node that represents an audit log
+    """
+
+    id = graphene.ID(source="pk", required=True)
+    metadata = GenericScalar()
+    resource_json_repr = GenericScalar()
+    resource_content_type = graphene.String()
```

Comment:
why?

Reply:
putting `required=True` on a graphene field makes it so that:
a) graphene will validate that they are never resolved to None
b) the schema will indicate that the value is always non-null, which simplifies client implementations since they don't need to check for this case

as far as i can tell these fields are always non-null so i think it makes sense to do so here

Reply:
these fields are on the `AuditLogNode` so it would just apply to querying if we updated them here